epub33/rs/index.html

```diff
@@ -1849,39 +1875,56 @@ <h4>Scripting Considerations</h4>
 					</li>
 				</ul>
 
-				<p>This specification recommends the following practices for handling untrusted scripts:</p>
-
+				<p>
+					In handling untrusted scripts this specification recommends that Reading Systems establish 
+					a unique <a href="https://url.spec.whatwg.org/#origin">origin</a> [[URL]] allocated to each 
+					<a>EPUB Publication</a> (see <a href="#sec-scripted-content"></a>). Adopting this
+					approach isolates publications from each other, thereby limiting access to cookies, DOM storage, etc. 
+					Examples of Web APIs that are tied to the concept of “origin” include Web Storage [[?WEBSTORAGE]] and 
+					IndexedDB [[?INDEXEDDB]], which EPUB content documents can interact with via scripting.
+					This specification recommends the following practices:
```

Comment:
This sentence kind of comes out of the blue and isn't clear why the practices are recommended.

Seems like it's missing an "in addition" and/or some connection to untrusted scripts.

Reply:
I have adopted your first proposal.

For the second, I actually did some more, separating the security/'origin' problems from the rest of the bullet items which were not really origin related but more privacy one. Can you have a look?

(If you are o.k. with the text, go ahead and merge it...)